joj_submitter/__main__.py

```diff
@@ -117,18 +193,53 @@ def submit(
         if no_wait:
             return
         res = self.get_status(response.url)
-        fore_color = Fore.RED if res[0] != "Accepted" else Fore.GREEN
+        if json_format:
+            print(json.dumps(res.to_dict()))
```

Comment:
logger?

Reply:
I think logger is for log. Print JSON in stdout will be more convenient for redirection and piping, so even this can be done with logger, I think use `print` will be a better choice.

Reply:
Everything is in stdout for now.